### PR TITLE
chore(invity): update @types/invity-api to version 1.0.8

### PR DESCRIPTION
**Pull request by yanascz**

### CODE REVIEW

File: packages/suite/src/services/suite/invityAPI.ts [hunk 1]
```diff
@@ -500,30 +68,6 @@ export type VerifySmsCodeResponse =
     | VerifySmsCodeInvalidResponse
     | VerifySmsCodeErrorResponse;
```
Comment (yanascz) [inline packages/suite/src/services/suite/invityAPI.ts]: @LukasRada, what about these? Are they missing in version 1.0.7 on purpose? Or maybe they are obsolete?

File: packages/suite/src/services/suite/invityAPI.ts [hunk 2]
```diff
@@ -27,453 +27,20 @@ import {
...
-    /** KYC process didn't start yet. */
-    | 'Open'
-    /** KYC process is in progress. Might take some time to resolve. */
-    | 'InProgress'
-    /** KYC process passed successfully. */
-    | 'Verified'
-    /** KYC docs are invalid or anything could be wrong. Expecting reason from our partner to handover to the user. */
-    | 'Failed'
-    /** KYC status check ended up in error state. */
-    | 'Error';
```
Comment (yanascz) [inline packages/suite/src/services/suite/invityAPI.ts:227]: @LukasRada, is this value missing in the DT version?

File: packages/suite/src/views/wallet/coinmarket/savings/overview/index.tsx [hunk 3]
```diff
@@ -123,7 +119,8 @@ function renderSavingsStatus(
     savingsFiatSum: string,
     savingsCryptoSum: string,
     savingsTrade?: SavingsTrade,
-    kycFinalStatus?: SavingsKYCStatus,
+    // TODO: Is Error supposed to be present in SavingsKYCStatus?
```
Comment (yanascz) [inline packages/suite/src/views/wallet/coinmarket/savings/overview/index.tsx:122]: CC @LukasRada

File: packages/suite/src/types/wallet/coinmarketSavingsSetupContinue.ts [hunk 4]
```diff
@@ -35,7 +35,8 @@ export type SavingsSetupContinueContextValues = Omit<
     maximumPaymentAmountLimit?: number;
     savingsTrade?: SavingsTrade;
     isSavingsTradeLoading: boolean;
-    kycFinalStatus?: SavingsKYCStatus;
+    // TODO: Is Error supposed to be present in SavingsKYCStatus?
```
Comment (yanascz) [inline packages/suite/src/types/wallet/coinmarketSavingsSetupContinue.ts:38]: CC @LukasRada

File: packages/suite/src/types/wallet/coinmarketSavingsPaymentInfo.ts [hunk 5]
```diff
@@ -9,6 +9,7 @@ export type SavingsPaymentInfoContextValues = SavingsContextValues & {
     handleSubmit: () => void;
     isSubmitting: boolean;
     copyPaymentInfo: (paymentInfoKey: keyof SavingsPaymentInfo) => void;
-    kycFinalStatus?: SavingsKYCStatus;
+    // TODO: Is Error supposed to be present in SavingsKYCStatus?
```
Comment (yanascz) [inline packages/suite/src/types/wallet/coinmarketSavingsPaymentInfo.ts:12]: CC @LukasRada

File: packages/suite/src/services/suite/invityAPI.ts (hunk 1 above)
Comment (yanascz) [reply packages/suite/src/services/suite/invityAPI.ts]: Removed, no longer necessary.

File: packages/suite/src/services/suite/invityAPI.ts (hunk 2 above)
Comment (yanascz) [reply packages/suite/src/services/suite/invityAPI.ts:227]: Fixed with DT version 1.0.8.

**Review by komret — APPROVED** [review]

**Pull request merged**